src/csharp.tmLanguage.yml

```diff
@@ -2509,6 +2511,18 @@ repository:
       '2': { name: keyword.preprocessor.error.cs }
       '3': { name: string.unquoted.preprocessor.message.cs }
 
+  preprocessor-load:
+    match: \b(load)\b\s*(\"[^"]*\")
```

Comment:
Yup, that'd be better. Be sure to set `end` to `(?=$)`. Take a look at `preprocessor-line` for an example.